packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -71,8 +71,8 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
     delayBeforeMouseOpen: 3600, // The approximate time the coachmark shows up
     width: 36,
     height: 36,
-    beaconColorOne: '#00FFEC',
-    beaconColorTwo: '#005EDD',
+    beaconColorOne: DefaultPalette.blueLight,
```

Comment:
use one of the theme* colors, since blueLight won't really change theme-to-theme (and will be very different than the themePrimary below in many themes)